### DISCUSSION TITLE
請問出現ffxiv_dx11.exe+5219EB要如何解決

### DISCUSSION THREAD

**Discussion by ssaammgithub** [GpointChen/FFXIVChnTextPatch-GP#434] [General]
你好，我之前使用漢化都正常，今天嘗試使用Reshade不太成功，移除Reshade後，就開始出現這個錯誤無法進入遊戲
![螢幕擷取畫面 2024-05-11 195941](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/169540418/e60ec6e0-8dd2-46e4-a6b7-8693acd75e0b)

我還原後可以進入遊戲，嘗試再漢化一樣會出現這個錯誤
也試過下載backup-20240430取代back，再漢化，也還是會出現這個錯誤
最後將ffxiv主程式移除後重新下載，再漢化，仍然出現這個錯誤

目前只能使用ffxiv-20240430直接覆蓋，才能進遊戲，但就沒有高清字體了
想請問這要如何解決，謝謝

**Comment by GpointChen** [comment]
漢化的時候在選項可以選擇只替換字體，你試試看？

**Reply by ssaammgithub** [reply]
我目前ffxiv-20240430直接覆蓋ffxiv主程式下的檔案，可以進遊戲
如果嘗試漢化只替換字體，是要選csv，然後目標語言選英文嗎?(我最初漢化是選英文)
還是目標語言要選日文？
謝謝

**Reply by GpointChen** [reply]
> 如果嘗試漢化只替換字體，是要選csv，然後目標語言選英文嗎?(我最初漢化是選英文)

只替換字體的話其他選項應該不影響，你這樣選應該沒問題。

**Reply by GpointChen** [reply]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/34565875/4b24014b-d41b-41a2-8e02-183e3493200a)

確定有取消勾選「替換文本」就行了

**Reply by GpointChen** [reply]
不過要注意如果你是先覆蓋後才用漢化器，漢化器的backup會備份到覆蓋後的檔案，所以你需要額外手動備份檔案。

**Reply by ssaammgithub** [reply]
剛剛用ffxiv-20240430覆蓋，然後照上面漢化替換字體，但還是出現同樣的錯誤

**Reply by GpointChen** [reply]
檢查一下UI介面設定有沒有改？

https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94

**Reply by ssaammgithub** [reply]
感謝，確實是UI介面沒有設定高解析度的問題
之前有改成一般，看看ui有什麼不同，因為沒差別就忘了改回來
抱歉，我的疏失讓你辛苦了
謝謝你的協助，也感謝你的漢化。
